packages/perseus/src/widgets/orderer/orderer.tsx

```diff
@@ -300,11 +295,6 @@ class Card extends React.Component<CardProps, CardState> {
     }
 }
 
-const NORMAL = "normal";
-const AUTO = "auto";
-const HORIZONTAL = "horizontal";
-const VERTICAL = "vertical";
```

Comment:
Two of these weren't used at all. Looking into it more it seemed like we could just use a string union.